lib/components/src/Zoom/ZoomIFrame.tsx

```diff
@@ -34,27 +31,19 @@ export default class ZoomIFrame extends Component<IZoomIFrameProps> {
     return false;
   }
 
-  componentWillUnmount() {
-    this.iframe.removeEventListener('load', this.setAttributeDataIsLoaded);
-  }
-
-  setAttributeDataIsLoaded() {
-    this.iframe.setAttribute('data-is-loaded', 'true');
-  }
-
   setIframeInnerZoom(scale: number) {
     try {
       if (browserSupportsCssZoom()) {
+        Object.assign(this.iframe.contentDocument.body.style, {
```

Comment:
iframe.contentDocument can't work cross origin.. but I'm guessing addressing that is out of scope.

can we use `useRef()` to get the current element instead though?